packages/perseus/src/widgets/interactive-graphs/axis-labels.tsx

```diff
@@ -0,0 +1,58 @@
+import {vec} from "mafs";
+import React from "react";
+
+import {getDependencies} from "../../dependencies";
+
+import {pointToPixel} from "./graphs/use-transform";
+import useGraphConfig from "./reducer/use-graph-config";
+
+export default function AxisLabels(props) {
+    const {range, labels, width, height} = useGraphConfig();
+
+    const yAxisLabelLocation: vec.Vector2 = [0, range[1][1]];
+    const xAxisLabelLocation: vec.Vector2 = [range[0][1], 0];
+
+    const [xAxisLabelText, yAxisLabelText] = labels;
+    const graphInfo = {
+        range,
+        width,
+        height,
+    };
+    const [x1, y1] = vec.add(
+        pointToPixel(xAxisLabelLocation, graphInfo)[0],
+        [0, 0],
+    );
+    const [x2, y2] = vec.add(
+        pointToPixel(yAxisLabelLocation, graphInfo)[0],
+        [0, -24],
+    );
+
+    const {TeX} = getDependencies();
+
+    return (
+        <>
+            <span
+                style={{
+                    position: "absolute",
+                    left: x1,
+                    top: y1,
+                    fontSize: "14px",
+                    transform: "translate(7px, -50%)",
+                }}
```

Comment:
I'm thinking that we don't need to calculate a position to place these labels. We should be able to do this entirely in CSS.

For the "x" label, the CSS would look like:
```
.x-axis-label {
    position: absolute;
    left: 100%;
    top: 50%;
    font-size: 14px;
    transform: translate(7px, -50%); /* assumes we want a fixed 7px spacing */
}
```
and the "y" label would look like:
```
.y-axis-label {
    position: absolute;
    left: 50%;
    font-size: 14px;
    transform: translate(-50%, calc(-100% - 7px));
}
```

If we decide to change the font size in the future, this will auto-adapt to the new size of the element. Another benefit is that we don't have to execute the two calls to `pointToPixel()`.

Reply:
Hmm, I think Ben mentioned not having both inline styles and CSS, so I moved everything to be inline. I suppose I could move everything to be on the css file instead. But doesn't this method require the axes to be like a plus sign? Sometimes the axes will be more of an "L" shape on the page, which I don't think would work with what you have here. I could be wrong though.

Reply:
I just realized that the CSS that I provided only works for centered labels. If we need to place the labels towards one edge of the graph, this won't work. Meaning that the call to `pointToPixel()` would still need to be made. While my preference is still to place the styling in CSS classes, and then adjust the position (if needed) by setting a CSS variable (via the `style` attribute), I think the main premise for my original recommendation has lost much of its allure. So, the current method is fine.

Reply:
Thank you for pointing this out though! Your recommendations are getting me thinking more about using CSS and trying to use a CSS file more. Personally, I prefer seeing everything in a CSS file, but I also prefer not using classes unless there are multiple instances of the class. I think that's not the preference generally though, so I defaulted to using the inline styling. Definitely appreciate your CSS reminders though :)